source/testnet/net/guides/run-node-macos.rst

```diff
@@ -0,0 +1,214 @@
+.. _run-node-macos:
+
+==============================
+Run and manage a node on macOS
+==============================
+
+.. contents::
+   :local:
+   :backlinks: none
+
+This guide describes how to run and manage a node on macOS. You can also run a
+node using :ref:`Docker <run-a-node>` for Linux distributions or :ref:`Ubuntu
+<run-node-ubuntu>`.
+
+
```

Comment:
Since this running a node on macOS is currently only available on testnet, we need to add a note (which will be removed for the mainnet release)
.. Note::
      Currently, running a node on macOS is only supported on the testnet.